evm/src/cpu/kernel/asm/handlers.asm

```diff
@@ -0,0 +1,4 @@
+global handle_stop:
```

Comment:
We can use the recently-added `stop` in place of this `handle_stop`.

I think we can also use `exception` in place of `handle_invalid_jump_dst`. I think an invalid jump would be handled in the same way as other exceptions like stack underflow/overflow.

(And of course we have the `exp` you wrote, though it's currently a "pure" function, so I guess we should add a syscall wrapper around it.)

Feel free to rename them though; maybe `handle_` (or `sys_` or something) is a good way to indicate that they're syscall / trap handlers.

Reply:
Will change!